src/phoenix/datasets/fixtures.py

```diff
@@ -414,8 +374,59 @@ def load_example(use_case: str) -> ExampleDatasets:
 
     """
     fixture = _get_fixture_by_name(use_case)
-    primary_dataset, reference_dataset = download_fixture_if_missing(use_case)
+    primary_dataset, reference_dataset = get_datasets(use_case)
     print(f"📥 Loaded {use_case} example datasets.")
     print("ℹ️ About this use-case:")
     print(fixture.description)
     return ExampleDatasets(primary=primary_dataset, reference=reference_dataset)
+
+
+class Metadata(NamedTuple):
+    path: str
+    mediaLink: str
+    md5Hash: str
+
+    def save_media(self, location: Path) -> Path:
+        data_file = location / self.path
+        md5_file = data_file.with_name(data_file.stem + ".md5")
+        data_file.parents[0].mkdir(parents=True, exist_ok=True)
+        if data_file.is_file() and md5_file.is_file():
+            with open(md5_file, "r") as f:
+                if f.readline() == self.md5Hash:
+                    return data_file
+        request.urlretrieve(self.mediaLink, data_file)
+        with open(md5_file, "w") as f:
+            f.write(self.md5Hash)
+        return data_file
+
+
+class GCS(NamedTuple):
```

Comment:
```suggestion
class GCSAsset(NamedTuple):
```
Better to be clear it's not just GCS :)